windows/RNSVG/GroupView.cpp

```diff
@@ -97,21 +96,40 @@ void GroupView::UpdateProperties(IJSValueReader const &reader, bool forceUpdate,
   }
 
   if (invalidate && SvgParent()) {
-    SvgRoot().InvalidateCanvas();
+    SvgRoot().Invalidate();
   }
 }
 
-void GroupView::CreateGeometry(UI::Xaml::CanvasControl const &canvas) {
-  auto const &resourceCreator{canvas.try_as<ICanvasResourceCreator>()};
-  std::vector<Geometry::CanvasGeometry> geometries;
-  for (auto const &child : Children()) {
+void GroupView::CreateGeometry() {
+  std::vector<ID2D1Geometry*> geometries;
```

Comment:
Clarifiying question on how this works. Is this taking a ref for each item, and if it's not is that okay?

Reply:
Looks like this still needs addressing. When we looked at it last you had a matching code comment you were going to duplicate here?